crates/containerd-shim-wasm/src/sandbox/shim.rs

```diff
@@ -1364,12 +1365,39 @@ where
                     continue;
                 }
 
-                let os_source = m.source().as_ref().unwrap().clone().into_os_string();
-                let source = os_source.to_string_lossy();
+                let dest = m.destination();
```

Comment:
Can we add more details in the commit message for this?
Maybe also a comment as to what we're doing?

Reply:
Update done a6ddbfba983df9ddf7e112a4d1c95ac6877187e1 and fe4f067d932b159af81f5ce02d20e7de935e8fbd.
In fact, I am not the original author, so I help to complete it from my memory and slack record of we discussed before. If anything I misunderstand, we could ask @rumpl help that.